public/javascripts/widgets/voip-tab.js

```diff
@@ -545,9 +552,14 @@ class VoipTab {
     if (!this.readable) {
       return '';
     }
-    return `<div class="edit-tab d-none" id="tab_voip-${this.deviceIndex}">` +
+    return `<div 
+        class="edit-tab ${this.isTabSelected ? '' : 'd-none'}"
+        id="tab_voip-${this.deviceIndex}"
+        data-permission="${this.permissions}"
+      >` +
```

Comment:
Como não cabe mais em uma linha, talvez fique mais organizado ter um array de atributos e trocar isso aqui por um join:
```suggestion
    const mainDivAttributes = [
      `class="edit-tab ${this.isTabSelected ? '' : 'd-none'}"`
      `id="tab_voip-${this.deviceIndex}"`
      `data-permission="${this.permissions}"`
    ];
    return `<div ${mainDivAttributes.join(' ')}>` +
```